.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+| Q | A
+| --- | ---
+| Bug? | no
```

Comment:
Why `no`? Different to a PR, where yes should enforce thinking about the answer, a no in the issue enforces thinking about it is really a bug (and if it's critical) or you only want to request a feature.